src/rustllvm/PassWrapper.cpp

```diff
@@ -424,6 +424,12 @@ extern "C" void LLVMRustPrintTargetFeatures(LLVMTargetMachineRef TM) {
   printf("Available features for this target:\n");
   for (auto &Feature : FeatTable)
     printf("    %-*s - %s.\n", MaxFeatLen, Feature.Key, Feature.Desc);
+  printf("Rust-specific features:\n");
```

Comment:
Sorry to keep nit-picking, but I think we should still have a blank line between the sections too.
```suggestion
  printf("\nRust-specific features:\n");
```

Reply:
alright, done